litmus-portal/frontend/src/components/SideBar/styles.ts

```diff
@@ -39,9 +39,9 @@ const useStyles = makeStyles((theme: Theme) => ({
     justifyContent: 'center',
     '&:hover': {
       backgroundColor: theme.palette.secondary.light,
-      color: theme.palette.getContrastText(theme.palette.primary.contrastText),
+      color: theme.palette.homePageCardBackgroundColor,
```

Comment:
https://github.com/litmuschaos/litmus/pull/2120/files#r496476443

Reply:
similar usage as mention in the above comment.
https://github.com/litmuschaos/litmus/pull/2120#discussion_r496723353